Task08/script.js

```diff
@@ -0,0 +1,60 @@
+(() => {
+  'use strict';
+
+  const getRecord = (event) => {
+		const apiURL = kintone.api.url('/k/v1/app/form/fields.json'); //https://…　ではなく、kintone.api.url() を使うことで短縮できる
+		const getParams = {
+			"app": 46,
```

Comment:
開いているアプリを指定するときは `kintone.app.getId()` でアプリIDを指定できます！
（コードを移行したときに書き換えないで済むのでこっちのほうが汎用性がありますね）

```suggestion
			"app": kintone.app.getId(),
```